### ISSUE TITLE
[Accessibility: Error Identification] Device Ids input field not getting focused on validation failure in Add ECKD device page. 

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/gingers390x#46]
In Add ECKD Device page, try to remove devices from black list without filling the device id field.
Error message displayed but input field is not getting focused.

![eckd device](https://cloud.githubusercontent.com/assets/13199397/13984567/0f3df928-f11d-11e5-8ca9-617b9264fe13.jpg)
